.codecov.yml

```diff
@@ -0,0 +1,38 @@
+codecov:
+  require_ci_to_pass: no
+
+comment: false
+
+component_management:
+  individual_components:
+    - component_id: module_beluga
+      name: beluga
+      paths:
+        - beluga/**
+      statuses:
+        - type: project
+          target: auto
+    - component_id: module_beluga_amcl
+      name: beluga_amcl
+      paths:
+        - beluga_amcl/**
+      statuses:
+        - type: project
+          target: auto
+          informational: true
```

Comment:
`beluga_amcl` coverage is low and will not block PRs, for now we keep it at an informational level because it is not trivial to add tests for that code. This will change soon.